<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coorconverttool</class>
 <widget class="QDockWidget" name="coorconverttool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>472</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>coorconverttool</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Microsoft Tai Le</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Coordinate to FOV</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Image Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Image Height:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QSpinBox" name="fovImageWidthspinBox">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>3264</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fovImageHeightspinBox">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>2448</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fovXspinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fovYspinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="fovpushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Microsoft PhagsPa</family>
          </font>
         </property>
         <property name="text">
          <string>Convert</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>FOV:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Angle:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="fovlineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fovAnglelineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Microsoft Tai Le</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Calculate Distance</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>X1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="disX1spinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Y1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="disY1spinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>X2:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="disX2spinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Y2:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="disY2spinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="dispushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Microsoft PhagsPa</family>
          </font>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Microsoft PhagsPa</family>
            </font>
           </property>
           <property name="text">
            <string>Distance(Pixel):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dislineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string notr="true">RGB ==&gt; Lab</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">R:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rgb2lab_RGB_RspinBox">
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="prefix">
              <string notr="true"/>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">G:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rgb2lab_RGB_GspinBox">
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="prefix">
              <string notr="true"/>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">B:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rgb2lab_RGB_BspinBox">
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="prefix">
              <string notr="true"/>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="RGB2LabpushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Microsoft PhagsPa</family>
          </font>
         </property>
         <property name="text">
          <string>==&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">L:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rgb2lab_Lab_LdoubleSpinBox">
             <property name="prefix">
              <string notr="true"/>
             </property>
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">A:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rgb2lab_Lab_adoubleSpinBox">
             <property name="prefix">
              <string notr="true"/>
             </property>
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-128.000000000000000</double>
             </property>
             <property name="maximum">
              <double>127.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">B:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rgb2lab_Lab_bdoubleSpinBox">
             <property name="prefix">
              <string notr="true"/>
             </property>
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-128.000000000000000</double>
             </property>
             <property name="maximum">
              <double>127.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
